components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Extensions/OpcConfigEx.cs

```diff
@@ -194,8 +194,8 @@ await ShowCertificateStoreInformationAsync(appConfig, logger)
                 int certNum = 1;
                 logger.Information("Rejected certificate store contains {count} certs.", certs.Count);
                 foreach (var cert in certs) {
-                    logger.Information($"{certNum++:D2}:" + " Subject '{subject}' (thumbprint: {thumbprint})",
-                        cert.Subject, cert.Thumbprint);
+                    logger.Information("{certNum:D2}:" + " Subject '{subject}' (thumbprint: {thumbprint})",
```

Comment:
Those two strings can be merged into `"{certNum:D2}: Subject '{subject}' (thumbprint: {thumbprint})"`.

Reply:
right